Tests/LibWeb/Text/expected/link-element-media-attribute.txt

```diff
@@ -1 +0,0 @@
-document background: rgb(0, 128, 0)
```

Comment:
Just as an example, this does not look correct, especially since the test specifies:
```js
        window.onload = function () {
            println("document background: " + getComputedStyle(document.body).backgroundColor);
        };
```

Reply:
Yeah I'm going through these, there's some tests that should be async, but aren't.